### PR TITLE
use Brewfile to avoid already installed warnings

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

**Review by DrewDennison — COMMENTED** [review]
Very cool!

**Comment by stephengroat** [comment]
seems like the ocaml opam cert is out of date, waiting until it works

Screen Shot 2020-05-06 at 7 52 44 PM

**Comment by DrewDennison** [comment]
@stephengroat yeah that just broke all our builds lol. Hopefully it gets fixed tomorrow.

**Comment by ievans** [comment]
@DrewDennison looks like this is passing, just can't push the docker tag to our registry from the fork.

**Comment by DrewDennison** [comment]
@stephengroat cert got fixed! I I'm happy to merge

File: .github/workflows/mac-build.yml [hunk 1]
```diff
@@ -10,8 +12,31 @@ jobs:
...
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Run OSX build
-        run: ./release-scripts/osx-release.sh
+      - name: Checkout submodules
+        shell: bash
+        run: |
+          # If your submodules are configured to use SSH instead of HTTPS please uncomment the following line
+          # git config --global url."https://github.com/".insteadOf "git@github.com:"
+          git remote set-url origin https://github.com/darlinghq/darling.git
```
Comment (brendongo) [inline .github/workflows/mac-build.yml:20]: Why are we setting url to darling?

File: .github/workflows/mac-build.yml [hunk 2]
```diff
@@ -10,8 +12,31 @@ jobs:
...
+          auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+          git submodule sync --recursive
+          git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+      - name: Install Brewfile
```
Comment (brendongo) [inline .github/workflows/mac-build.yml:24]: We should just call the osx-release script.

**Review by brendongo — CHANGES_REQUESTED** [review]
See comments above

**Comment by brendongo** [comment]
Closing due to inactivity. @stephengroat feel free to reopen.

**Pull request closed without merging**